.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- Please read [CONTRIBUTING.md](https://github.com/grote/Transportr/blob/master/CONTRIBUTING.md) before submitting your pull request -->
+
+## Context/Motivation
+
+<!-- Provide a link to the corresponding issue if applicable or explain the context -->
+
+## Screenshots/Videos
+
+<!-- If applicable -->
+
+## Tests
+
+<!-- Explain how you tested your development -->
+
+- Step 1
+- Step 2
+- Step ...
+
+## Checklist
+- [ ] Changes has been tested on an Android device or Android emulator with API 21
+- [ ] Pull request includes screenshots or videos if the changes impact UI
+- [ ] `connectedCheck` passes
```

Comment:
What do you mean by `connectedCheck`? Maybe some further explanation would be nice.